lib/OpenQA/WebAPI/Controller/API/V1/Job.pm

```diff
@@ -321,18 +321,18 @@ sub destroy {
 
 =over 4
 
-=item prio()
+=item niceness()
 
-Sets priority for a given job.
+Sets niceness for a given job.
 
 =back
 
 =cut
 
-sub prio {
+sub niceness {
     my ($self) = @_;
     return unless my $job = $self->find_job_or_render_not_found($self->stash('jobid'));
-    my $res = $job->set_prio($self->param('prio'));
+    my $res = $job->set_niceness($self->param('niceness'));
```

Comment:
Does this makes sense though? If we keep `prio` as a fallback, we might as well just not change it at all. It would just make everything more confusing.